lib/webrtc/rtcpeerconnection/index.js

```diff
@@ -5,6 +5,8 @@ if (typeof webkitRTCPeerConnection !== 'undefined') {
   module.exports = require('./chrome');
 } else if (typeof mozRTCPeerConnection !== 'undefined') {
   module.exports = require('./firefox');
+} else if (typeof navigator !== 'undefined' && navigator.userAgent.match(/Version\/(\d+).(\d+)/)) {
```

Comment:
@markandrus We should also make these checks independent of the prefixed `PeerConnection` constructors.